### ISSUE TITLE
Provide better feedback in the intervals exercise.

### ISSUE THREAD

**Issue by eight0153** [icantthinkofacreativeusername/ourcontributiontoblindmusicians#15]
When a user sings an interval, they should be at least shown the notes they sang. They could also be shown  some measure of how far the notes they sang were from the target notes (for example, distance in semitones or cents).

**Comment by Jmannn** [comment]
Yes, for a disability support idea, they should be told how close and have their pitchs played againts the target
